src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
+import java.util.Iterator;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+public class Tagging {
+	
+	private String type;
+	private Tag tag;
+	
+	/*
+	 * Constructor
+	 */
+	public Tagging(String type,  Tag tag){
+		if(type == "ADD"){
+			this.type ="+ ";
+		}
+		else if (type == "REMOVE"){
+			this.type ="- ";
+		}
+		this.tag = tag;
+	}
+	
+	public String getType(){
+		return type;
+	}
+	
+	public String getTag(){
+		return tag.toString();
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)